python/cli/prelude_cli/templates/template.go

```diff
@@ -2,7 +2,7 @@
 ID: $ID
 NAME: $NAME
 UNIT: $UNIT
-CREATED: $CREATED
+CREATED: $TIME
```

Comment:
It's probably worth updating the template/test in the javascript test code too (just to keep it all consistent)
* https://github.com/preludeorg/libraries/blob/master/javascript/sdk/test/sdk.test.ts#L154
* https://github.com/preludeorg/libraries/blob/master/javascript/sdk/test/templates/template.go#L5

Reply:
done!